test/package1/package1VersionCreate.test.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as os from 'os';
+import { instantiateContext, MockTestOrgData, restoreContext, stubContext } from '@salesforce/core/lib/testSetup';
+import { Duration } from '@salesforce/kit';
+import { expect } from 'chai';
+import { Connection, Lifecycle } from '@salesforce/core';
+import { assert } from 'sinon';
+import { package1VersionCreate } from '../../src/package1';
+import { PackagingSObjects } from '../../src/interfaces';
+
+const options = {
+  MetadataPackageId: '0HD4p000000blVyGAI',
+  VersionName: 'Test',
+  Description: 'Test',
+  MajorVersion: 0,
+  MinorVersion: 0,
+  IsReleaseVersion: false,
+  ReleaseNotesUrl: 'Test',
+  PostInstallUrl: 'Test',
+  Password: 'Test',
+};
+
+const successResult = {
+  Status: 'SUCCESS',
+  Id: '0HD4p000000blVyGAI',
+  MetadataPackageVersionId: '04t4p000002Bb4lAAC',
+  MetadataPackageId: '03346000000MrC0AAK',
+};
+
+const queuedResult = {
+  Status: 'QUEUED',
+  Id: '0HD4p000000blVyGAI',
+  MetadataPackageVersionId: '04t4p000002Bb4lAAC',
+  MetadataPackageId: '03346000000MrC0AAK',
+};
+
+describe('Package1 Version Create', () => {
+  const $$ = instantiateContext();
+  const testOrg = new MockTestOrgData();
+  let conn: Connection;
+  let sobjectStub: sinon.SinonStub;
+
+  beforeEach(async () => {
+    stubContext($$);
+    await $$.stubAuths(testOrg);
+    conn = await testOrg.getConnection();
+    sobjectStub = $$.SANDBOX.stub(conn.tooling, 'sobject')
+      .onFirstCall()
+      .returns({
+        // @ts-ignore - to avoid stubbing every property of sobject
```

Comment:
jsyk, ts-sinon handles this nicely.  I think it's `stubInterface` or something like that.